simon.sh

```diff
@@ -192,12 +197,10 @@ put_descr() {
 put_cursor_after_prompt() {
     local xoffset
     xoffset="$(printf -- '%s' "$VBUF" | tail -1 \
-        | tr -d "$B$BLD$RST" | wc -m)"
+        | eval '$SED -E "s/\x1B(\[[0-9;]*[JKmsu]|\(B)//g"' | wc -c)"
```

Comment:
```suggestion
        | eval '$SED -E "s/\x1B(\[[0-9;]*[JKmsu]|\(B)//g"' | wc -m)"
```
wc is intended to count characters, not bytes